encord/client.py

```diff
@@ -255,10 +261,12 @@ def __init__(
         self,
         querier: Querier,
         config: Config,
+        api_client: ApiClient,
```

Comment:
don't see any reason for the pattern of parameters to be different from what 'project client' has. Should be optional and `None` when it's via and api key